packages/manager/src/features/Managed/ManagedLandingContent.tsx

```
@@ -139,7 +147,17 @@ export const ManagedLandingContent: React.FC<CombinedProps> = props => {
           exact
           strict
           path={`${props.match.path}/monitors`}
-          component={Monitors}
+          render={() => (
+            <Monitors
+              credentials={credentials.data}
+              loading={
+                (credentials.loading && contacts.lastUpdated === 0) ||
+                (contacts.loading && contacts.lastUpdated === 0)
+              }
+              groups={groups}
+              errorFromProps={credentials.error || contacts.error || undefined}
```

This bombs the Monitors view entirely if credentials or contacts can't be retrieved. You don't need these for displaying the table, but the create and edit actions won't work properly without them. We could try more complicated logic for error handling (show the table but disable the things) or just swallow the errors---users could still input their groups manually, and neither field is required.